### PR TITLE
Collection value updating

### LINKED ISSUE #6: Add key value editing for collection based values
currently value editing is only available for string based keys. We should allow editing of values for collections such as lists, sets, zsets, hashes

### PR DESCRIPTION
**Pull request by yassi**
resolves #6 

## Summary by CodeRabbit

- **New Features**
  - Add/edit/delete elements for lists, sets, sorted sets, and hashes via the key detail UI with per-item inline controls, add-forms, confirmation prompts, i18n, and permission-aware disabling.

- **Style**
  - New inline-edit controls and buttons with dark-mode tweaks; adjusted success message color.

- **Refactor**
  - Key detail UI migrated to a class-based view with unified request handling and consistent pagination metadata.

- **Tests**
  - Comprehensive add/edit/delete test suites covering pagination, permissions, edge cases, and data validation.

- **Docs / Chores**
  - Updated docs, README, Makefile, CI flows, and package version bump.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,44 +5,53 @@ PYPI_REPO ?= pypi   # can be 'testpypi' or 'pypi'
...
+	@echo "All packages in requirements.txt uninstalled"
+	@echo "Note that some dependent packages may still be installed"
+	@echo "To uninstall all packages, run 'pip freeze | xargs pip uninstall -y'"
+	@echo "Do this at your own risk. Use a python virtual environment always."
 
 test_install: build
 	pip uninstall -y $(PACKAGE_NAME) || true
-	pip install dist/*.whl
+	pip install -e .
 	python -c "import dj_redis_panel; print('✅ Import success!')"
```
[the remark this answers is not in the document]
Comment (yassi) [reply Makefile]: mkdocs is already in the existing requirements.txt

also switched to using `python -m` invocations in the latest push
